.NET/Microsoft.Recognizers.Text.Number/Arabic/Extractors/NumberRangeExtractor.cs

```diff
@@ -88,5 +100,200 @@ public NumberRangeExtractor(INumberOptionsConfiguration config)
         internal sealed override Regex AmbiguousFractionConnectorsRegex { get; }
 
         protected sealed override string ExtractType { get; } = Constants.SYS_NUMRANGE;
+
+        public override List<ExtractResult> Extract(string source)
```

Comment:
This looks like the same base NumberRangeExtractor in other languages. Can't the code be changed to re-use?
Ideally we should introduce specific language-specific customization points, but avoid duplicating code. Otherwise code will diverge and issues would need to be fixed in multiple places.

Reply:
Agreed, will be fixing that  and will update the same PR.